weaponlookup.js

```diff
@@ -0,0 +1,53 @@
+//making a weapon look up class for fe-heroes.
+//similar to the charlookup so gonna try to inherit everything from it
+var cLook=require('./characterlookup'); 
+
+
+module.exports = {
+	init(database){
+		init(database);
+	},
+	lookUp(object, callback){
+		return lookUp(object, callback)
+	}
+	
+
+}
+
+var db;//database variable
+
+
+function init(database){
+	console.log("I have been awakened");

```

Comment:
Let's leave this console.log() here but specify which class is being initiated. (This will be good for determining which classes were successfully initiated).